packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -387,8 +387,11 @@ export class SerializedStateManager {
 			this.mc.logger,
 			{
 				eventName: "getPendingLocalState",
-				notifyImminentClosure: props.notifyImminentClosure,
-				processedOpsSize: this.processedOps.length,
+				details: {
+					...props,
```

Comment:
i don't think everything on props is serializable

Reply:
Yeah, it's not - I'm erasing the thing that's not on the next line.

But, not very future proof.  I can revert.

Reply:
i'm fine with the move to details, but would rather keep it explicit